packages/sns/candid/sns_governance_test.idl.js

```diff
@@ -755,6 +769,11 @@ export const idlFactory = ({ IDL }) => {
         [GetMetadataResponse],
         ['query'],
       ),
+    'get_metrics' : IDL.Func(
+        [GetMetricsRequest],
+        [GetMetricsResponse],
+        ['composite_query'],
```

Comment:
Is that `composite_query` by agent-js? Or the frontend cannot call this function anyway?